### PR TITLE
Example for Readme

### PR DESCRIPTION
**Pull request by sjanssen2**
Adding instructions to run the moving pictures example

### CODE REVIEW

**Comment by sjanssen2** [comment]
@wasade @mortonjt since you are familiar with the topic, could you give this extension a quick review?

File: README.md [hunk 1]
```diff
@@ -21,3 +21,26 @@ A fragment may be reasonable to insert into multiple locations. However, downstr
...
+qiime fragment-insertion sepp-16s-greengenes \
+  --i-representative-sequences rep-seqs.qzv \
+  --o-tree insertion-tree.qza \
+  --o-placements insertion-placements.qza \
+  --o-classification insertion-taxonomy.qza
+```
+Output artifacts:
+   - `insertion-tree.qza`: ~[view]()~ | [download](https://github.com/biocore/q2-fragment-insertion/raw/readme_example/Example/insertion-tree.qza)
+   - `insertion-placements.qza`: ~[view]()~ | [download](https://github.com/biocore/q2-fragment-insertion/raw/readme_example/Example/insertion-placements.qza)
+   - `insertion-taxonomy.qza`: ~[view]()~ | [download](https://github.com/biocore/q2-fragment-insertion/raw/readme_example/Example/insertion-taxonomy.qza)
```
Comment (mortonjt) [inline README.md]: why are the view links crossed out?

Comment (sjanssen2) [reply README.md]: because I don't know how to set them up correctly :-/

Comment (sjanssen2) [reply README.md]: I keep getting an `TypeError: NetworkError when attempting to fetch resource.` :-(

**Comment by sjanssen2** [comment]
Hey @antgonza can you give this PR another shot?

File: README.md (hunk 1 above)
Comment (sjanssen2) [reply README.md]: Strange. In Chrome the same link lead to a different error message: `TypeError: Failed to fetch`

**Comment by antgonza** [comment]
we are gonna merge and then fix the links once the files are available

**Pull request merged**